modules/widgets/milestone/milestone.php

```diff
@@ -213,20 +197,28 @@ function widget( $args, $instance ) {
 		echo '<span class="date">' . esc_html( date_i18n( __( 'F jS, Y', 'jetpack' ), $milestone ) ) . '</span>';
 		echo '</div>';
 
-		if ( 1 > $diff ) {
-			/* Milestone has past. */
+		if ( ( 1 > $diff ) && ( 'until' === $type ) ) {
 			echo '<div class="milestone-message">' . $instance['message'] . '</div>';
 		} else {
+			if ( 'since' === $type ) {
+				$text = 'ago.';
+			} else {
+				$text = 'to go.';
+			}
+
 			/* Countdown to the milestone. */
-			echo '<div class="milestone-countdown">' . sprintf( __( '%1$s %2$s to go.', 'jetpack' ),
-				'<span class="difference">' . esc_html( $number ) . '</span>',
-				'<span class="label">' . esc_html( $label ) . '</span>'
+			echo '<div class="milestone-countdown">' . sprintf( __( '%1$s %2$s %3$s', 'jetpack' ),
```

Comment:
We would need a translator comment here to know what each placeholder stands for, otherwise this is impossible to translate.